### PR TITLE
[TIMOB-20228] iOS: Set minimum Xcode version to 7.1

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-20228

### CODE REVIEW

**Comment by hansemannn** [comment]
Travis now succeeds until the Xcode 7.x API's that break the build process. As soon as travis runs with Xcode 7.1 or later, it will succeed and we can merge it. /cc @cheekiatng

File: iphone/Classes/ContactsModule.m [hunk 1]
```diff
@@ -352,13 +347,12 @@ -(void)showContacts:(id)args
 	RELEASE_TO_NIL(selectedPersonCallback)
 	RELEASE_TO_NIL(selectedPropertyCallback)
 	RELEASE_TO_NIL(picker)
-#if IS_XCODE_7
     RELEASE_TO_NIL(contactPicker)
-#endif
-	cancelCallback = [[args objectForKey:@"cancel"] retain];
+
+    cancelCallback = [[args objectForKey:@"cancel"] retain];
```
Comment (AngelkPetkov) [inline iphone/Classes/ContactsModule.m]: indentation

File: iphone/Classes/ContactsModule.m [hunk 2]
```diff
@@ -778,15 +770,15 @@ -(void)removePerson:(id)arg
...
 	if([TiUtils isIOS9OrGreater]) {
 		TiContactsPerson *person = arg;
 		saveRequest = nil;
 		saveRequest = [person getSaveRequestForDeletion];
 		return;
 	}
-#endif
-	[self removeRecord:[arg record]];
+
+    [self removeRecord:[arg record]];
```
Comment (AngelkPetkov) [inline iphone/Classes/ContactsModule.m]: Indentation

File: iphone/Classes/TiContactsGroup.m [hunk 3]
```diff
@@ -227,8 +226,8 @@ -(NSArray*)sortedMembers:(id)value
 			return nil;
 		}
 	}
-#endif
-	int sortType = [value intValue];
+
+    int sortType = [value intValue];
```
Comment (AngelkPetkov) [inline iphone/Classes/TiContactsGroup.m]: Indentation

File: iphone/Classes/TiContactsPerson.m [hunk 4]
```diff
@@ -587,8 +579,8 @@ -(TiBlob*)image
 		}
 		return nil;
 	}
-#endif
-	CFDataRef imageData = ABPersonCopyImageData([self record]);
+
+    CFDataRef imageData = ABPersonCopyImageData([self record]);
```
Comment (AngelkPetkov) [inline iphone/Classes/TiContactsPerson.m]: Indentation

File: iphone/Classes/TiContactsPerson.m [hunk 5]
```diff
@@ -613,8 +605,8 @@ -(void)setBirthday:(NSString*)date
 		person.birthday = [cal components:unitFlags fromDate:saveDate];
 		return;
 	}
-#endif
-	ABRecordSetValue([self record], kABPersonBirthdayProperty, (CFDateRef)[TiUtils dateForUTCDate:date], NULL);
+
+    ABRecordSetValue([self record], kABPersonBirthdayProperty, (CFDateRef)[TiUtils dateForUTCDate:date], NULL);
```
Comment (AngelkPetkov) [inline iphone/Classes/TiContactsPerson.m]: Indentation  ? Not sure if this one needs  to move more to the right or not

File: iphone/Classes/TiContactsPerson.m [hunk 6]
```diff
@@ -712,8 +704,8 @@ -(void)setValue:(id)value forUndefinedKey:(NSString*)key
 	}
 
 	id property = nil;
-#if IS_XCODE_7
-	if ([TiUtils isIOS9OrGreater]) {
+
+    if ([TiUtils isIOS9OrGreater]) {
```
Comment (AngelkPetkov) [inline iphone/Classes/TiContactsPerson.m]: Indentation of either the if or the contexts

**Comment by AngelkPetkov** [comment]
FR and CR looks good! Will merge ones Indentation is addressed.

**Pull request merged**